build/storage/tests/ptf_scripts/test_setup.py

```diff
@@ -0,0 +1,150 @@
+        self.setup_storage = TestPlatform(SSHTerminal(address=storage_address, user=user_name, password=password))
+
+    def runTest(self):
+        return_values = TestPlatform.setup_on_both_machines(
+            self.setup_storage.check_kvm,
+            self.setup_proxy.check_kvm,
+            timeout=30
+        )
+
+        assert return_values == (True, True)
+
+    def tearDown(self):
+        pass
+
+
+class TestSetupDockerCompose(BaseTest):
+
+    def setUp(self):
+        self.setup_proxy = TestPlatform(SSHTerminal(address=ipu_storage_address, user=user_name, password=password))
+        self.setup_storage = TestPlatform(SSHTerminal(address=storage_address, user=user_name, password=password))
+
+    def runTest(self):
+        return_values = TestPlatform.setup_on_both_machines(
+            self.setup_storage.setup_docker_compose,
+            self.setup_proxy.setup_docker_compose,
+            timeout=60,
+        )
+
+        assert return_values == (True, True)
+
+    def tearDown(self):
+        pass
+
+
+class TestSetupLibguestfsTools(BaseTest):
+
+    def setUp(self):
+        self.setup_proxy = TestPlatform(SSHTerminal(address=ipu_storage_address, user=user_name, password=password))
+        self.setup_storage = TestPlatform(SSHTerminal(address=storage_address, user=user_name, password=password))
+
+    def runTest(self):
+        return_values = TestPlatform.setup_on_both_machines(
+            self.setup_storage.setup_libguestfs_tools,
+            self.setup_proxy.setup_libguestfs_tools,
+            timeout=30,
+        )
+
+        assert return_values == (True, True)
+
+    def tearDown(self):
+        pass
+
+
+class TestInstalled(BaseTest):
+
+    def setUp(self):
+        self.setup_proxy = TestPlatform(SSHTerminal(address=ipu_storage_address, user=user_name, password=password))
+        self.setup_storage = TestPlatform(SSHTerminal(address=storage_address, user=user_name, password=password))
+
+    def runTest(self):
+        return_values = TestPlatform.setup_on_both_machines(
+            self.setup_storage.is_installed,
+            self.setup_proxy.is_installed,
+            timeout=60,
+        )
+
+        assert return_values == ([0, 0, 0], [0, 0, 0])
+
+    def tearDown(self):
+        pass
+
+
+class TestCheckSecurityPolicies(BaseTest):
+
+    def setUp(self):
+        self.setup_proxy = TestPlatform(SSHTerminal(address=ipu_storage_address, user=user_name, password=password))
+        self.setup_storage = TestPlatform(SSHTerminal(address=storage_address, user=user_name, password=password))
+
+    def runTest(self):
+        return_values = TestPlatform.setup_on_both_machines(
+            self.setup_storage.check_security_policies,
+            self.setup_proxy.check_security_policies,
+            timeout=30,
+        )
+
+        assert return_values == (True, True)
+
+    def tearDown(self):
+        pass
```

Comment:
It's first intergration tests but where is unittests - we should also test separately method from Terminal and TestPlatform